src/k8s-engine/wsl.ts

```diff
@@ -619,6 +627,34 @@ export default class WSLBackend extends events.EventEmitter implements K8s.Kuber
     }
   }
 
+  protected async installCACerts(): Promise<void> {
```

Comment:
I would add a comment that this doesn't require tarring up all the certs like for lima because WSL doesn't have the same limit on the number of parallel connections as a linux VM on macos